.env.template

```diff
@@ -28,7 +28,7 @@ SECRET_KEY_FILE=
 # ---------------------------------------------------------------
 
 # your default timezone See https://timezonedb.com/time-zones for a list of timezones
-TIMEZONE=Europe/Berlin
+TZ=Europe/Berlin
```

Comment:
is there a reason for changing this? thousands of people will have to change their environment ...

Reply:
ok so it can be taken from the default environment, that makes sense. should we in this case also comment out the TZ line in the default .env as the system should then load the hosts computer default time zone or do you think that many people might have their data center in a different time zone?

also added a comment below

Reply:
the alpine container still needs the TZ env variable passed.